### PR TITLE
try to improve codecov

### PR DESCRIPTION
**Pull request by gaborbernat**
If we can't use coveralls; we might need to try to fix it. Here it just retries if fails.

### CODE REVIEW

**Comment by gaborbernat** [comment]
Seems codecov does not support Azure DevOps. oh well codecov it is then.

**Comment by obestwalter** [comment]
if-at-first-you-dont-succeed-try-try-and-try-again

**Comment by obestwalter** [comment]
oh ... what? I thought coveralls does not play nicely yet with Azute DevOps. So both are not really there yet? Classic early adopter problems I guess.

File: azure-run-tox-env.yml [hunk 1]
```diff
@@ -32,5 +32,14 @@ jobs:
     - script: 'python -m tox -e coverage'
       displayName: generate coverage report
 
-    - script: 'python -m tox -e codecov -- -t $(CODECOV_TOKEN) --required -n "$(agent.os)-$(python.version)" --build "$(Build.DefinitionName)" --env OS=$(agent.os) pyt+hon=$(python.version)'
+    - script: |
+        for i in 1 2 3; do
```
Comment (thedrow) [inline azure-run-tox-env.yml]: You can use `travis_retry` instead of the shell script

Comment (gaborbernat) [reply azure-run-tox-env.yml]: Note there's no plan to use travis also. This is Azure DevOps, not Travis CI.

**Pull request closed without merging**